keps/prod-readiness/sig-multicluster/3335.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 3335
+alpha:
+  approver: "johnbelamaric"
```

Comment:
I would be happy to take this once this has SIG-level approval.

Reply:
Given that Wojtek already did that, this needs changing per his comment, still.